components/progressTracker/ProgressTracker.js

```diff
@@ -0,0 +1,47 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import Box from '../box';
+import ProgressStep from './ProgressStep';
+
+class ProgressTracker extends PureComponent {
+  render() {
+    const { color, children, activeStep } = this.props;
+
+    const classNames = cx(theme['progress-tracker']);
+
+    return (
+      <Box className={classNames}>
+        {React.Children.map(children, (child, index) => {
+          const currentActiveStep = activeStep < 0 ? 0 : activeStep;
+
+          return React.cloneElement(child, {
+            active: index === currentActiveStep,
+            completed: index < currentActiveStep,
```

Comment:
Also thinking about this completed: What if we want to have a 4-step procedure and step 5 means everything's finished, then we have shouldn't put step 5 as active but immediately finished, this forces us to set the step to 6 in order to have it as `completed`? Should we provide an option to force completed?

Reply:
I could add a prop called "allStepsAreCompleted" and then check for that too?

Reply:
I think that wouldn't be a luxury :)